sql_metadb/derived_tables/holdings_administrative_notes.sql

```diff
@@ -24,4 +24,15 @@ CREATE INDEX ON holdings_administrative_notes (administrative_note);
 
 CREATE INDEX ON holdings_administrative_notes (administrative_note_ordinality);
 
+COMMENT ON COLUMN Instance_id IS 'Inventory instances identifier';
+
+COMMENT ON COLUMN holdings_id IS 'the unique ID of the holdings record; UUID';
+
+COMMENT ON COLUMN holdings_hrid IS 'the human readable ID, also called eye readable ID. A system-assigned sequential ID which maps to the Instance ID';
+
+COMMENT ON COLUMN administrative_note IS 'Administrative notes';
+
+COMMENT ON COLUMN administrative_note_ordinality IS 'Administrative note ordinality';
```

Comment:
Please add the tablename in front of the attributes and change the name of 'instance_id' at the beginning with a lower case.
`holdings_administrative_notes.instance_id`

The semicolon on line 29 was not a problem when running in DBeaver. Maybe it would be better to find another sign.